challenges/3-area-trapezio/php/yzpeedro/challenge.php

```diff
@@ -0,0 +1,22 @@
+<?php
+	
+	//Recebe valores do terminal e coloca dentro de suas respectivas variáveis
+	$bMaior = $argv[1];
+	$bMenor = $argv[2];
+	$altura = $argv[3];
+
+	//Verifica se o valor inserido é númerico
+	if (!is_numeric($bMaior) || !is_numeric($bMenor) || !is_numeric($altura)) {
+
+		//retorna relatório de erro
+	    echo PHP_EOL . 'Por favor, insira um valor numérico!' . PHP_EOL;
+	    return false;
+
+	}else{
```

Comment:
Lembrar de espaçar as chaves entre o else do mesmo jeito que você faz com o if
```suggestion
	} else {
```